.github/workflows/ci.yaml

```diff
@@ -101,7 +101,8 @@ jobs:
 
   install-chart:
     name: install-chart
-    runs-on: ubuntu-20.04
+    runs-on:
+      group: retool-helm-kind-runners
```

Comment:
Lint and Test Charts / install-chart (v1.26.4) 

was otherwise failing 

![Uploading Screenshot 2023-09-08 at 10.53.30 AM.png…]()